Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -131,6 +141,10 @@ public function apply(ResourceInterface $resource, QueryBuilder $queryBuilder, R
                 }
 
                 $strategy = null !== $this->properties ? $this->properties[$property] : self::STRATEGY_EXACT;
+                if (strpos($strategy, 'i') === 0) {
+                    $strategy = substr($strategy, 1);
+                    $this->caseSensitive = false;
```

Comment:
Seems we already discussed this point https://github.com/dunglas/DunglasApiBundle/pull/384#discussion_r49444177.  Putting `caseSensitive` to a local variable also means adding a new argument to the `addWhereByStrategy` and to every `wrapCase` function, which seems to much. Let me know if you stick by it I'd implement it. I'm resetting the variable to true in my next commit.